<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QRadioButton" name="rb_up">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>69</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>rb_up</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rb_down">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>69</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>rb_down</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_box"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_pBarUp">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pBar_bar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>329</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
